src/commcare_cloud/terraform/modules/r53-private-zone-create-update/main.tf-backup

```diff
@@ -0,0 +1,14 @@
+resource "aws_route53_zone" "elastic-cache-dns" {
```

Comment:
This file is called `main.tf-backup` which makes me think this wasn't intended to be committed. Is that right?

Reply:
Yes @dannyroberts. I have deleted it.